qt/bundle.h

```diff
@@ -40,6 +40,11 @@ class APPSTREAMQT_EXPORT Bundle {
         Bundle& operator=(const Bundle& bundle);
         bool operator==(const Bundle& r) const;
 
+        /**
+         * \returns the internally stored AsBundle
+         */
+        _AsBundle *asBundle() const;
```

Comment:
Are these really necessary? I would argue that this is only needed in case the Qt/C++ interface is incomplete and someone therefore wants to use the C library directly. In that case, we should rather fix the Qt binding to include the missing stuff, and not give library users the means to hack around it and not report the issue.